app/javascript/pages/Dashboard/index.js

```diff
@@ -117,6 +117,7 @@ const milestoneLabelStyling = (item, milestone, user) => {
   return R.join(' ', classes)
 }
 
+// TODO: remove; sorting should be done on server side
```

Comment:
if we want this done, please create an issue for it.

Reply:
Found an existing relevant issue. I've left a comment. https://github.com/zendesk/volunteer_portal/issues/70#issuecomment-455379844